docs/Installation/Installation_Guide.rst

```diff
@@ -492,29 +492,28 @@ parameters as appropriate.)
    ::
 
     ingress:
-     enabled: "true"
-     annotations:
-       nginx.ingress.kubernetes.io/proxy-body-size: 0
-
-    hosts:
-      -  zenko.local
+      enabled: "true"
+      annotations:
+        nginx.ingress.kubernetes.io/proxy-body-size: 0
+      hosts:

```

Comment:
Is `hosts` supposed to be a sub-item of `ingress`? or is it on the same level as `ingress`

Reply:
It's supposed to go under ingress. That's the actual fix.